packages/compass/src/main/auto-updater.ts

```diff
@@ -0,0 +1,45 @@
+import { EventEmitter } from 'events';
+import { createLoggerAndTelemetry } from '@mongodb-js/compass-logging';
+import type { AutoUpdater, FeedURLOptions } from 'electron';
+import { autoUpdater as electronAutoUpdater } from 'electron';
+import got from 'got';
+
+const { debug } = createLoggerAndTelemetry('COMPASS-AUTO-UPDATES');
+
+/**
+ * Electron can't currently provide a consistent upgrade path for linux users.
+ * Instead, provide a stub class for linux so the auto update service is
+ * still called which allows us to know how many Linux users exist.
+ */
+class LinuxAutoUpdater extends EventEmitter implements AutoUpdater {
```

Comment:
Can we throw away this? or do we want to use it somehow? The premise of this comment doesn't make any sense at this point, we have other ways to count Linux users. 

Should we be showing a different dialog window to Linux users? I'd lean to delete this (we have some ticket around) and re-evaluate the experience independently.

Reply:
Having a stub for linux that is a no-op still makes sense to me so that I can just use autoUpdater directly without thinking about what platform I am on in the actual manager. Cleaning up the actual fetch and the comment sounds good to me